tests/integration/test_database_iceberg/test.py

```diff
@@ -305,6 +305,8 @@ def test_select(started_cluster):
         node.query(f"SELECT count() FROM {CATALOG_NAME}.`{namespace}.{table_name}`")
     )
 
+    assert int(node.query("SELECT count() FROM system.iceberg_history").strip()) == 1
```

Comment:
I think the fails are due to other tables

```suggestion
    assert int(node.query("SELECT count() FROM system.iceberg_history WHERE table_name = '{TABLE_NAME}").strip()) == 1
```